two-factor.php

```diff
@@ -66,7 +66,7 @@ function wpcom_vip_should_force_two_factor() {
 		return false;
 	}
 
-	if ( Two_Factor_Core::is_user_using_two_factor() ) {
+	if ( apply_filters( 'wpcom_vip_is_current_user_using_two_factor', false ) ) {
```

Comment:
Does the filter name need to be exactly the same name as the new function? Maybe drop the `current_` bit in the filter name?

Reply:
Sure, all set now